subsys/net/ip/tcp.h

```diff
@@ -137,8 +140,10 @@ struct net_tcp {
 	 * of various timing issues when timer is scheduled to run.
 	 */
 	u32_t ack_timer_cancelled : 1;
+	/* Tells if fin timer has been already cancelled. */
+	u32_t fin_timer_cancelled : 1;
 	/** Remaining bits in this u32_t */
-	u32_t _padding : 12;
+	u32_t _padding : 11;
```

Comment:
Lets deal with that in later patches, I do not want to delay this patch for too long.